composer.json

```
@@ -53,7 +53,7 @@
         "doctrine/dbal":                        "^2.10",
         "doctrine/doctrine-module":             "^4.0",
         "doctrine/orm":                         "^2.6.4",
-        "doctrine/persistence":                 "^1.3",
+        "doctrine/persistence":                 "^1.3.3|^2.0",
```

@TomHAnderson Looks good to me, but I am unsure why you are requiring doctrine/persistance ^1.3.3. The forward compatibility has been added with 1.3.0 already, see https://github.com/doctrine/persistence/tree/1.3.0/lib/Doctrine. Besides, according to https://getcomposer.org/doc/articles/versions.md#version-range you should the a double pipe.

```suggestion
        "doctrine/persistence":                 "^1.3 || ^2.0",
```